optics-core/optics-core.cabal

```diff
@@ -11,15 +11,23 @@ library
   hs-source-dirs: src
 
   build-depends: base >= 4.9 && < 5
+               , array >= 0.5.1.1 && < 0.6
+               , containers >= 0.5.8.1 && < 0.7
```

Comment:
Based on https://ghc.haskell.org/trac/ghc/wiki/Commentary/Libraries/VersionHistory we probably want `>= 0.5.7.1`

Reply:
I see, we need this for `Seq.foldlWithIndex`, thus we can either narrow the GHC versions or add a shim.